src/Satooshi/Bundle/CoverallsBundle/Api/Jobs.php

```diff
@@ -96,7 +96,7 @@ public function collectEnvVars(array $env)
 
         try {
             $this->jsonFile->fillJobs($envCollector->collect($env));
-        } catch (\Satooshi\Bundle\CoverallsV1Bundle\Entity\Exception\RequirementsNotSatisfiedException $e) {
+        } catch (\Satooshi\Bundle\CoverallsBundle\Entity\Exception\RequirementsNotSatisfiedException $e) {
```

Comment:
:+1: for doing it, but not manually. let us do it later with PHP CS Fixer
(after Windows related branch gonna be merged)